src/org/montclairrobotics/sprocket/pipeline/Pipeline.java

```diff
@@ -0,0 +1,34 @@
+package org.montclairrobotics.sprocket.pipeline;
+
+import java.util.ArrayList;
+
+public class Pipeline <T>{
```

Comment:
pipeline.get() runs it. They are not a standalone process, they run when they are requested. This can be changed as it doesn't fit with our strategy for the rest of Sprocket, but in this instance I think it is simpler and fits with what we are trying to do. Also easier to ensure which steps happen when.